README.md

```diff
@@ -1,91 +1,48 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
-## Getting Started
-
-This repository includes files with plain SQL that can be used to recreate a database:
-
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-
 <a name="readme-top"></a>
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Kindly make sure that you delete the `Microverse` badge from your README file, instead, you can add your custom badge or `logo` to make your repo more professional. You are indeed a student of an esteemed online web development school but the goal of the `Microverse` is to create a professional developer who knows how to document the work properly.